### DISCUSSION TITLE
Install to Another Drive

### DISCUSSION THREAD

**Discussion by oppej** [WinMerge/winmerge#2527] [Q&A] [answered]
I must be missing something - the WinMerge manual says...

"Start the installer executable (usually named WinMerge-version-setup.exe).
Click Next in the License Agreement screen.
In the Select Destination Location page, accept the location for WinMerge or, if you choose another location, we recommend that you install WinMerge in its own folder. Click Next."

The "Select Destination Location page" does not appear for me (see screenshots). All 4 installation options end up at the same place without giving me a choice of drive.

![Screenshot 2024-11-09 171747](https://github.com/user-attachments/assets/53afb0a2-4ce4-4667-a672-574e56dc1b77)
![Screenshot 2024-11-09 171830](https://github.com/user-attachments/assets/93ed7825-c30d-451a-a601-3abfc6b8f91e)
![Screenshot 2024-11-09 171902](https://github.com/user-attachments/assets/2afb4b03-060c-4d7f-a6c3-5b338ce4f1f5)
![Screenshot 2024-11-09 171930](https://github.com/user-attachments/assets/187b425c-d629-44f9-a501-6bc4f9bf797f)

Hopefully someone can tell me what's going on

Cheers

**Comment by sdottaka** [comment]
Please uninstall WinMerge first, then run the WinMerge installer and you will be able to choose the destination folder for the installation.

**Comment by oppej** [comment, marked as answer]
Thanks sdottaka,
Now I understand why the installer would not give me a destination folder option. I hadn't uninstalled the previous C drive installation.
Appreciate your help
